index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Movielist</title>
+    <link rel="stylesheet" href="./StyleSheets/Navigation.css" />
+    <link rel="stylesheet" href="./StyleSheets/normalize.css" />
+  </head>
+  <body>
+    <div class="container">
+      <aside class="nav">
+        <figure class="logo">
+          <img class="LogoImg Desktop" src="./images/Logo.svg" alt="" />
+          <div class="LogoImg TabletAndMobile"></div>
+        </figure>
+
+        <nav class="menu">
+          <menu class="pages">
+            <div class="search menuPages">
+              <img src="./images/Search.svg" alt="" class="iconimg" />
+              <input type="text" class="searchinput" placeholder="Search" />
+            </div>
+
+            <a href="" class="menuPages">
+              <div class="menuPages">
+                <img src="./images/home.svg" alt="" class="iconimg" />
+                <p>Home</p>
+              </div>
+            </a>
+
+            <a href="" class="menuPages">
+              <div class="menuPages">
+                <img src="./images/earth.svg" alt="" class="iconimg" />
+                <p>Library</p>
+              </div>
+            </a>
+
+            <a href="" class="menuPages">
+              <div class="menuPages">
+                <img src="./images/Friends.svg" alt="" class="iconimg" />
+                <p>Friends</p>
+              </div>
+            </a>
```

Comment:
Can we use a loop here?

Just keep the variables in an object and run it over a loop so that we can avoid this duplication.